<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ar::MainWindow</class>
 <widget class="QMainWindow" name="ar::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>582</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Diabetes Diagnosis</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="diagnosis_tab">
       <attribute name="title">
        <string>Diagnosis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>User Information</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="line_name"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="line_address"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Age</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinbox_age">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Gender</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="combobox_gender">
               <item>
                <property name="text">
                 <string>Male</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Female</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Pregnant</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="checkbox_pregnant">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Weight (kg)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="dspinbox_weight"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Height (cm)</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="spinbox_height"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Medical Data</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_pregnancies">
               <property name="text">
                <string>Pregnancies</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinbox_pregnancies">
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
               </property>
               <property name="accelerated">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Glucose</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinbox_glucose"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Blood Pressure</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinbox_blood"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Skin Thickness</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinbox_skin"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Insulin</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="spinbox_insulin"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Body Mass Index</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="dspinbox_bmi">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Diabetes Pedigree Function</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="dspinbox_dpf">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Family Diseases</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Grandparents</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinbox_grandparents"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Parents</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinbox_parents"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Siblings</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinbox_siblings"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_start_diagnosis">
            <property name="text">
             <string>Start Diagnosis</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="result_tab">
       <attribute name="title">
        <string>Result</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="result_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data_tab">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
